src/lib/airtable/request.js

```diff
@@ -124,57 +123,37 @@ export const createManyCustomerUpdates = async (records) => {
   return Promise.all(createPromises);
 };
 
-// NONGENERATED: Create a meter reading for a customer
```

Comment:
Can you revert this file? Your PR shouldn't have changes to `request.js` or `schema.js`